sql/connect/client/jdbc/src/main/scala/org/apache/spark/sql/connect/client/jdbc/SparkConnectDatabaseMetaData.scala

```diff
@@ -199,164 +162,114 @@ class SparkConnectDatabaseMetaData(conn: SparkConnectConnection) extends Databas
 
   override def getCatalogSeparator: String = "."
 
-  override def supportsSchemasInDataManipulation: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsSchemasInDataManipulation: Boolean = true
 
-  override def supportsSchemasInProcedureCalls: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsSchemasInProcedureCalls: Boolean = true
```

Comment:
Databricks JDBC v1.0.10-oss returns `false`, but Spark SQL does not have such limitation